raiden-derive/src/ops/get.rs

```diff
@@ -11,6 +12,18 @@ pub(crate) fn expand_get_item(
     let client_name = format_ident!("{}Client", struct_name);
     let builder_name = format_ident!("{}GetItemBuilder", struct_name);
     let from_item = super::expand_attr_to_item(&format_ident!("res_item"), fields, rename_all_type);
+    let project_expression = fields.named.iter().map(|f| {
+        let ident = &f.ident.clone().unwrap();
+        let renamed = crate::finder::find_rename_value(&f.attrs);
+        format!("#{}", create_renamed(ident.to_string(), renamed, rename_all_type))
+    }).collect::<Vec<String>>().join(", ");
+    let expression_attribute_names_str = fields.named.iter().map(|f| {
+        let ident = &f.ident.clone().unwrap();
+        let renamed = crate::finder::find_rename_value(&f.attrs);
+        let result = create_renamed(ident.to_string(), renamed, rename_all_type);
+        format!("\"#{}\": \"{}\"", result.clone(), result)
+    }).collect::<Vec<String>>().join(", ");
```

Comment:
Thanks for your great work :)
It feels a bit confusing and I don't think we need to depends on `serde` for this use.

I think it is enough to expand hashmap like following.
```suggestion
    let expanded_expression_attribute_names = {
        let insertion = fields.named.iter().map(|f| {
            let ident = &f.ident.clone().unwrap();
            let renamed = crate::finder::find_rename_value(&f.attrs);
            let result = create_renamed(ident.to_string(), renamed, rename_all_type);
            quote! {
                names.insert(
                    format!("#{}", #result.clone()),
                    #result.clone().to_string(),
                );
            }
        });
        quote! {
            let mut names: std::collections::HashMap<String, String> = std::collections::HashMap::new();
            #(#insertion)*
        }
    };
```